tests/backup/backup_helper.go

```diff
@@ -6364,6 +6364,14 @@ func dumpMongodbCollectionOnConsole(kubeConfigFile string, collectionName string
 // validateCRCleanup validates CR cleanup created during backup or restore
 func validateCRCleanup(resourceInterface interface{},
 	ctx context.Context) error {
+
+	log.InfoD("Validating CR cleanup")
+
+	if GetClusterProviders()[0] == "gke" {
+		log.Infof("Skipping CR cleanup validation in case of GKE")
+		return nil
+	}
```

Comment:
Add a `TODO` to remove this code once we figure out how get get the correct kubeconfig in case of GKE

Reply:
Done